charts/katalyst/charts/colocation/orm_values.yaml

```diff
@@ -0,0 +1,69 @@
+global:
+  # Overrides all components' image by global image
+  image: {}
+
+# Overrides katalyst-agent values
+katalyst-agent:
+  enabled: true
+
+  image:
+    registry: cr-pro-cn-guilin-boe.cr.volces.com
+    repository: wz-test/katalyst-agent
```

Comment:
Do we need to switch the `registry` and `repository` to DockerHub? @WangZzzhe @pendoragon

Reply:
> Do we need to switch the `registry` and `repository` to DockerHub? @WangZzzhe @pendoragon

I fix it in https://github.com/kubewharf/charts/pull/15, pls have a review